src/modules/Tasks.js

```diff
@@ -0,0 +1,57 @@
+class Task {
+  constructor(description, index, completed = false) {
+    this.description = description;
+    this.index = index;
+    this.completed = completed;
+  }
+
+  static LoadList = (tasksList) => {
+    const todolist = document.querySelector('.list');
+    let string = '';
+    tasksList.forEach((element) => {
+      string += `<li class="taskContainer">
+      <input type="checkbox" class="checkbox" id="c${element.index}">
+      <input type="text" name="i${element.index}" id="${element.index}" class="task" value="${element.description}" data-type="task">
+      <button id="${element.index}" class="delete"></button>
+    </li>`;
+    });
+    todolist.innerHTML = string;
+    const checkboxes = document.querySelectorAll('input[type="checkbox"]');
+    const tasks = document.querySelectorAll('input[data-type="task"]');
+    tasksList.forEach((element) => {
+      checkboxes[element.index - 1].checked = element.completed;
+      if (element.completed === true) {
```

Comment:
- [x] Excellent job applying this conditional here, however, I think it would be better to use `if (element.completed)` instead of `if (element.completed === true)` since it is the same and it helps you keep your code simple (KISS).